Makefile

```diff
@@ -58,3 +64,22 @@ test_unit:
 test_integration: all
 	$(CRYSTAL) run test/integration/*_test.cr -- --parallel=1
 
+.PHONY: install
+install: install-bin install-man
+
+.PHONY: install-bin
+install-bin: bin/shards
+	mkdir -p $(BINDIR)
+	cp bin/shards $(BINDIR)/
+
+.PHONY: install-man
+install-man:
+	mkdir -p $(MANDIR)/man1 $(MANDIR)/man5
+	cp $(filter %.1,$(MANPAGES)) $(MANDIR)/man1/
+	cp $(filter %.5,$(MANPAGES)) $(MANDIR)/man5/
+
+.PHONY: uninstall
+uninstall:
+	rm -f "$(BINDIR)/shards"
+	rm -f "$(MANDIR)/man1/$(notdir $(filter %.1,$(MANPAGES)))"
```

Comment:
Shouldn't this be `$(addprefix $(MANDIR)/man1/,$(notdir $(filter %.1,$(MANPAGES))))`?

Reply:
Uh, yes, mea culpa. Fixed now.